typescript/core/src/v0/Mcap0IndexedReader.test.ts

```diff
@@ -548,4 +548,280 @@ describe("Mcap0IndexedReader", () => {
       message6,
     ]);
   });
+
+  it("correctly reads overlapping chunks", async () => {
```

Comment:
making the description a bit more specific. as is, it's basically the same as the previous test description
```suggestion
  it("uses stable sort when loading overlapping chunks", async () => {
```